Remote id info - subedited (#3404)
PX4/PX4-user_guide: en/peripherals/remote_id.md

The specification requires that UAVs broadcast data such as: real-time location/altitude, serial number, operator ID/location, status, etc.

PX4 works with Remote ID modules that target the FAA [standard Remote ID rules](https://www.faa.gov/uas/getting_started/remote_id).
These are designed to be integrated into the vehicle, and broadcast Open Drone ID messages (Open Drone ID is an open source implementation of Remote ID) using id, position, and other information that is supplied by an autopilot.
The "standard rules" modules enable less restrictive operation than "broadcast rules" modules, which are standalone modules with an integrated GPS that do not have any communication with the autopilot.

## Supported Hardware

PX4 integrates with Remote ID hardware that supports:

- [Open Drone ID](https://mavlink.io/en/services/opendroneid.html) MAVLink protocol<Badge type="tip" text="PX4 v1.14" />
- Remote ID over CAN<Badge type="tip" text="PX4 main (v1.16)" />

>   ::: warning

Author: FYI I moved your warning down here from the top to be a little less confronting - to easy to misread as all remote ids not supported except in main.

Reviewer: This is from the original Remote ID Documentation that @julianoes wrote. I just pulled in those changes so that I didn't write over or rewrite any of the information that was already present.

Author: :-) All good!

>   DroneCAN Remote IDs hardware is only supported on `main` branch builds (builds _after_ PX4 v1.15).

Author: ```suggestion
  Remote ID hardware can only be connected via DroneCAN on `main` branch builds (builds _after_ PX4 v1.15).
```

  - PX4 v1.14 does not yet receive `OPEN_DRONE_ID_ARM_STATUS`.
- `OPEN_DRONE_ID_ARM_STATUS` must be forwarded to the GCS, if present for additional error reporting.
- [OPEN_DRONE_ID_BASIC_ID](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_BASIC_ID) specifies a serial number in an invalid format (not ANSI/CTA-2063 format).
- The vehicle ID is expected to be tamper resistant.

[PX4-Autopilot/21647](https://github.com/PX4/PX4-Autopilot/pull/21647) is intended to address the known issues.

## Troubleshooting

### CAN Remote ID Not Working

::: info
This information was tested with the CAN Cube ID from CubePilot.
It _should_ also apply to CAN Remote ID modules from other vendors.
:::

To confirm that the Remote ID is working:

- Check that the Remote_ID node appears on the UAVCAN list.

  Run the following command in the [QGroundControl MAVLink Console](http://localhost:5173/px4_user_guide/en/debug/mavlink_shell.html#qgroundcontrol-mavlink-console):

  ```sh
  uavcan status
  ```

  The remote id should appears in the list as shown below (there maybe other nodes in the list too).

  ```plain
  Online nodes (Node ID, Health, Mode):

>      125 OK         OPERAT

Author: Is this really what you see? Does that mean we're looking for a node with id 125? I.e. how do you know this node is a remote ID node - in particular if there are multiples?

Reviewer: So Enter UAVCAN generates a lot of text that I didn't want to put all of it making the document too long. When only the Cube ID is attached then yes this is the only node that is present but there can be other nodes attached.

Author: That's cool, but it doesn't quite confirm what I need. Let me put it another way - is 125 always the node of any remote id? 

Otherwise, i I think you are saying that there is no way to tell if a remote id is present unless you diff what the list looks like before and after.

Reviewer: The safest approach would be to diff between the before and after on the CAN bus. Not sure if all Remote ID modules will have 125 as there Node ID. I was only able to test out the Cube ID.

Author: Thanks @Aaron1356 . I have edited this to put the message check first - if that works you don't need to do anything. Then for the `uavcan status` check I have suggested a simple node count as first option, and differencing as a better check. I think this is OK.

  ```

- Check that the `OPEN_DRONE_ID_BASIC_ID` and `OPEN_DRONE_ID_LOCATION` messages appear in the QGroundControl [MAVLink Inspector](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/analyze_view/mavlink_inspector.html) (QGC **Analyze Tools > MAVLink Inspector**).

If the above are not true, then the Remote ID itself may need to be configured:

1. Open QGroundControl
2. Navigate to the [Application settings](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/settings_view/general.html): **Application Settings > General > Miscellaneous**.

> 3. Select `Enable Remote ID`.

Reviewer: Well to be more clear it is in the miscellaneous section of the general tab, but yes there is a checkbox to enable remote ID on QGC which will then enable the Remote ID tab so that data like operator ID, self ID, and Basic ID can be set for the drone and ground station.

> 3. Select `Enable Remote ID`.

Author: @Aaron1356 Hmmm. On my QGC daily I have a dedicated tab as shown below. What version are you using?

Either way, it sounds like we send them to this page depending on the version of QGC used.

![image](https://github.com/user-attachments/assets/8a8f0642-0178-4f93-a38a-cfd94ec2841c)

> 3. Select `Enable Remote ID`.

Author: I think I'll just note some of the options
